opencensus/trace/span.h

```diff
@@ -142,7 +157,7 @@ class Span final {
   bool IsRecording() const;
 
  private:
-  Span() {}
+  Span();
```

Comment:
Why do we have this? Please comment if this is needed why.

Reply:
Deleted.